manifests/dependencies.pp

```diff
@@ -5,14 +5,38 @@
     'gcc', # Needed for compiling Python modules
     'git', # Needed for pulling the MISP code and those for some dependencies
     'zip', 'mariadb',
-    'python-devel', 'python2-pip', 'python-lxml', 'python-dateutil', 'python-six', # Python related packages
+    'python-lxml', 'python-dateutil', 'python-six', # Python related packages
     'libxslt-devel', 'zlib-devel',
     'rh-php56', 'rh-php56-php-fpm', 'rh-php56-php-devel', 'rh-php56-php-mysqlnd', 'rh-php56-php-mbstring', 'php-pear', 'rh-php56-php-xml', 'rh-php56-php-bcmath', # PHP related packages
     'php-mbstring', #Required for Crypt_GPG
     'haveged',
     'sclo-php56-php-pecl-redis', # Redis connection from PHP
     'php-pear-crypt-gpg', # Crypto GPG 
+    'python-magic', # Advance attachment handler
+    'ssdeep', 'ssdeep-libs', 'ssdeep-devel', #For pydeep
   ],
     { 'ensure' => 'present' }
   )
+
+  class { '::python' :
```

Comment:
this should probably be handled in a profile, or misp should only include it.

Reply:
I can't include the python class as python dev needs to be `present` (the default in the class is set to `absent`), needed for the installation of some of the MISP dependencies. As such I've introduced a class parameter called `$manage_python`. Those that are managing python somewhere else can simply set it to false.